Update CrystalConf 2023 into retrospective + blog post (#520)
crystal-lang/crystal-website: _posts/2023-11-09-memories-from-crystalconf.md

---
title: Memories from CrystalConf
author: straight-shoota
summary: Impressions from the conference in Berlin
---

CrystalConf 2023 was a sounding success with a terrific lineup of speakers and talks.
We spent two great days in Berlin sharing knowledge, ideas, and beers.

> Recordings of the talks are available on YouTube.

Reviewer: If we want to send this soon, we should mention the list is being constructed.

```suggestion
Recordings of the talks are being uploaded to a playlist on YouTube.
```

<a href="https://youtube.com/playlist?list=PLt-CsM4G1WoadONHl3zPN_Ts5PqH8TgMZ" class="btn btn-large btn-flat">Video playlist</a>

<figure markdown="1">
  ![](/assets/events/crystalconf2023/audience.jpg)

>   <caption>The audience listens to Johannes' talk about noticing little details that point towards deficiencies.</caption>

Reviewer: Is it just me? The many indirections makes this hard to parse.

```suggestion
  <caption>The audience listens to Johannes' talk about deficiencies in programs that might come from hidden little details.</caption>
```

</figure>

<figure markdown="1">
  ![](/assets/events/crystalconf2023/panel.jpg)

>   <caption>In the panel discussion we talked the future of Crystal with the audience. We also had the immense pleasure to share the stage with two core team alumni.</caption>

Reviewer: Do we mention anything of Stephie being back?

Author: I simply shortened the caption.

>   <caption>In the panel discussion we discussed the future of Crystal with the audience. We also had the immense pleasure to share the stage with two core team alumni.</caption>

Reviewer 2: ```suggestion
  <caption>In the panel discussion we talked the future of Crystal with the audience. We also had the immense pleasure to share the stage with two core team alumni.</caption>
```
